airflow/operators/python_operator.py

```diff
@@ -145,3 +158,196 @@ def execute(self, context):
             self.skip(context['dag_run'], context['ti'].execution_date, downstream_tasks)
 
         logging.info("Done.")
+
+class PythonVirtualenvOperator(PythonOperator):
+    """
```

Comment:
For the docstring to get picked up you'll need an autoclass entry in:
https://github.com/apache/incubator-airflow/blob/master/docs/code.rst